nbed/ham_builder.py

```diff
@@ -444,8 +457,10 @@ def build(
             core_indices = np.array(core_indices)
             active_indices = np.array(active_indices)
 
+        """
```

Comment:
is unrestricted tapering implemented?

Reply:
I think me and @TimWeaving agreed that symmer would handle this ok but maybe he could confirm?

Reply:
@tom-bickley @TimWeaving any update?